tests/alerts_test.go

```diff
@@ -26,24 +26,27 @@ import (
 	"github.com/openshift/pagerduty-short-circuiter/pkg/pdcli"
 )
 
+// incident retuns a pagerduty incident object with pre-configured data.
 func incident(incidentID string) pdApi.Incident {
 	return pdApi.Incident{
 		Id: incidentID,
 	}
 }
 
```

Comment:
What is the purpose this wrapper function `incident` ? If you notice it doesn't do anything in addition, other than calling `pdApi.Incident({Id: incidentID})`, may be we can get substitute the call directly and get rid of this function.

Reply:
I see this as a clean way of creating a pd incident object through a simple function call.